psutil/_pslinux.py

```diff
@@ -1017,23 +1017,24 @@ def disk_io_counters():
     system as a dict of raw tuples.
     """
     # determine partitions we want to look for
+    partition_suffix = re.compile(r'p*\d+')
```

Comment:
Wouldn't this still match nvme0n1? It is looking for 0 or more 'p's followed by one or more digits.

Reply:
@kportertx 
The `partition_suffix` is used to match partition suffix, not the whole partition name. Certainly, it will be better to write `r'^p*\d+$'` to avoid confusing. I will change it in the new commit.